### ISSUE TITLE
How to disable missing-doc-suite warning?

### ISSUE THREAD

**Issue by 4nif** [MarketSquare/robotframework-robocop#769]
### What happened?

Hello

How can we disable warning `missing-doc-suite` warning from the `settings.json`?
I have `.resource` file and contains only `*** Variables ***` 
When I enable robocop lint in `settings.json` VSCode shown warning `missing-doc-sute`

I have try to add `# robocop: disable=missing-doc-suite` after `*** Variables ***` and warning still appear

Is it possible to disable this warning on the specific resource file?

### Operating System

Mac

### Robocop version

2.6.0

**Comment by bhirsz** [comment]
Did you try to put ``# robocop: disable=missing-doc-suite`` in the first line of the file? If you put it after ``*** Variables ***`` it will surely not be recognized (since it will disable the rule for all lines after the disabler definition). 

So:

```
# robocop: disable=missing-doc-suite

*** Variables ***
${ENV}    dev
```

**Comment by mnojek** [comment]
@4nif Has your problem been resolved by @bhirsz's suggestion, or do you still need our support?
